iDiff/cmd/iDiff_test.go

```diff
@@ -0,0 +1,31 @@
+package cmd
+
+import (
+	"testing"
+)
+
+type testpair struct {
+	values   []string
+	expected bool
+}
+
+var tests = []testpair{
+	{[]string{}, false},
+	{[]string{"one"}, false},
+	{[]string{"one", "two"}, true},
+	{[]string{"one", "two", "three"}, true},
+	{[]string{"one", "two", "three", "four"}, false},
+}
+
+func TestArgNum(t *testing.T) {
+	for _, test := range tests {
+		valid, err := checkArgNum(test.values)
+		if valid != test.expected {
+			if test.expected {
+				t.Errorf("Expected error but got none")
```

Comment:
the valid boolean is supposed to indicate whether the arguments are valid or not, so if they don't match the expected validity of the test, that's when there's an error in the test... but you're right, I accidentally flipped the statements, so I'll fix that.